scripts/package-rpm.sh

```diff
@@ -29,15 +29,6 @@ PACKAGE_VERSION="$("$PROJECT_ROOT/scripts/version.sh")"
 echo "Packaging .rpm for $ARCHIVE_NAME"
 echo "TARGET: $TARGET"
 
-#
-# Safeguard
-#
-
-if [[ "$UID" == "0" ]]; then
-  echo "Error: aborting RPM build due to execution as root" >&2
-  exit 1
-fi
```

Comment:
I'm pretty sure this was here for a reason... :smile: Is this intended, and not something that slipped in after, say, temporary local changes, etc?

Reply:
The restriction on running RPM build as root (damage to the underlying system) isn't relevant in a container world.